src/foremast/elb/format_listeners.py

```diff
@@ -177,13 +178,30 @@ def generate_custom_cert_name(env='', account='', certificate=None):
     # TODO: Investigate moving this to a remote API, then fallback to local file if unable to connect
     try:
         rendered_template = get_template(template_file='infrastructure/iam/tlscert_naming.json.j2', **template_kwargs)
+        tlscert_dict = json.loads(rendered_template)
     except ForemastTemplateNotFound:
         LOG.info('Unable to find TLS Cert Template...falling back to default logic...')
         return cert_name
 
     try:
-        cert_name = json.loads(rendered_template)[env][certificate]
+        LOG.info("Attempting to find TLS Cert using TLS Cert Template v1 lookup...")
+        cert_name = tlscert_dict[env][certificate]
+        LOG.info("Found TLS certificate named %s under %s using TLS Cert Template v1", certificate, env)
     except KeyError:
-        LOG.error("Unable to find TLS certificate named %s under %s in TLS Cert Template", certificate, env)
+        LOG.error("Unable to find TLS certificate named %s under %s using v1 TLS Cert Template.", certificate, env)
+
+    if cert_name is None:
```

Comment:
You should probably do `if not cert_name`

Reply:
We discussed offline on this.
@jvasallo felt more comfortable being explicit on checking the value of `cert_name` was `None`. The concern was around if somehow `cert_name` earlier in the function gets a value, it will be match against this condition.
I think its a fair concerns, so we are leaving it as-is.